File: samples/client/3_1_0_unit_test/java/src/main/java/org/openapijsonschematools/client/components/schemas/StringTypeMatchesStrings.java
package org.openapijsonschematools.client.components.schemas;
import org.checkerframework.checker.nullness.qual.Nullable;
import org.openapijsonschematools.client.schemas.StringJsonSchema;

public class StringTypeMatchesStrings {
    // nest classes so all schemas and input/output classes can be public
    
    
    public static class StringTypeMatchesStrings1 extends StringJsonSchema {
        private static @Nullable StringTypeMatchesStrings1 instance = null;
        public static StringTypeMatchesStrings1 getInstance() {
            if (instance == null) {
                instance = new StringTypeMatchesStrings1();
            }
            return instance;
        }
    }

}

File: samples/client/3_1_0_unit_test/kotlin/src/main/kotlin/org/openapijsonschematools/client/components/schemas/StringTypeMatchesStrings.kt
package org.openapijsonschematools.client.components.schemas
import org.openapijsonschematools.client.schemas.StringJsonSchema

class StringTypeMatchesStrings: StringJsonSchema() {
    // nest classes so all schemas and input/output classes can be public
    
    
    class StringTypeMatchesStrings1 : StringJsonSchema.StringJsonSchema1() {
        companion object {
            @Volatile
            private var instance: StringTypeMatchesStrings1? = null
    
            fun getInstance() =
                instance ?: synchronized(this) {
                    instance ?: StringTypeMatchesStrings1().also { instance = it }
                }
        }
    }

}
